src/commands_controller.ts

```diff
@@ -13,19 +13,23 @@ export class CommandsController implements Disposable, NeovimExtensionRequestPro
 
     public constructor(private main: MainController) {
         this.disposables.push(
-            vscode.commands.registerCommand("vscode-neovim.ctrl-f", () => this.scrollPage("page", "down")),
-        );
-        this.disposables.push(
-            vscode.commands.registerCommand("vscode-neovim.ctrl-b", () => this.scrollPage("page", "up")),
-        );
-        this.disposables.push(
-            vscode.commands.registerCommand("vscode-neovim.ctrl-d", () => this.scrollPage("halfPage", "down")),
-        );
-        this.disposables.push(
-            vscode.commands.registerCommand("vscode-neovim.ctrl-u", () => this.scrollPage("halfPage", "up")),
+            commands.registerCommand("vscode-neovim.ctrl-f", () => this.scrollPage("page", "down")),
+            commands.registerCommand("vscode-neovim.ctrl-b", () => this.scrollPage("page", "up")),
+            commands.registerCommand("vscode-neovim.ctrl-d", () => this.scrollPage("halfPage", "down")),
+            commands.registerCommand("vscode-neovim.ctrl-u", () => this.scrollPage("halfPage", "up")),
+            commands.registerCommand("vscode-neovim.ctrl-e", () => this.scrollLine("down")),
+            commands.registerCommand("vscode-neovim.ctrl-y", () => this.scrollLine("up")),
+            eventBus.on("reveal", ([at, updateCursor]) => this.revealLine(at, !!updateCursor)),
+            eventBus.on("move-cursor", ([to]) => this.goToLine(to)),
+            eventBus.on("scroll", ([by, to]) => this.scrollPage(by, to)),
+            eventBus.on("scroll-line", ([to]) => this.scrollLine(to)),
+            eventBus.on("insert-line", async ([type]) => {
```

Comment:
So the main change in this PR is to remove `NeovimExtensionRequestProcessable` in favor of `eventBus.on` ? It's helpful to mention that kind of key idea in the PR description / commit message.